apps/codeforafrica/src/lib/data/common/navigation.js

```diff
@@ -0,0 +1,19 @@
+async function getNavigation(api) {
+  const navigation = await api.findGlobal("navigation");
+  const {
+    logo: { image },
+    menus = [],
+    socialLinks = [],
+  } = navigation;
+
+  return {
+    logo: {
+      alt: image?.alt || "Logo",
+      src: image?.url || "/images/cfa-logo.svg",
+    },
+    menu: menus?.map(({ label, href }) => ({ content: label, href })),
```

Comment:
```suggestion
    menus: menus.map(({ label, href }) => ({ content: label, href })),
```
1. Since it's a list of `menu`, then `menus` make sense, no?
2. We've already default `menus = []`, no?